src/core/main.c

```diff
@@ -698,6 +699,7 @@ static int parse_config_file(void) {
                 { "Manager", "DefaultTasksAccounting",    config_parse_bool,             0, &arg_default_tasks_accounting          },
                 { "Manager", "DefaultTasksMax",           config_parse_tasks_max,        0, &arg_default_tasks_max                 },
                 { "Manager", "CtrlAltDelBurstAction",     config_parse_emergency_action, 0, &arg_cad_burst_action                  },
+                { "Manager", "ResctrlOptions",            config_parse_string,           0, &arg_resctrl_options                   },
```

Comment:
All file systems are configured through /etc/fstab: generally systemd will have a set of default options, so they mount point does not have to be explicitly configured, but will apply the options from /etc/fstab if the fs is defined there. See https://www.freedesktop.org/wiki/Software/systemd/APIFileSystems/. This has the advantage that it's discoverable for users (everybody knows about fstab), and the configuration is not siloed in a custom config file and can be accessed by other programs.

Please don't add a custom setting for this.